.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -65,6 +65,9 @@ steps:
       - label: ":windows: x-pack/winlogbeat Win 2019 Unit Tests"
         key: "mandatory-win-2019-unit-tests"
         command: |
+          Push-Location -Path .buildkite/scripts
+          ./install_sysmon.ps1
```

Comment:
Nit: We can pre-install sysmon to the base Windows image(when the PR is merged) We need to raise an issue for that

Reply:
100%, I was also thinking maybe keeping our own copy since there is no way (that I know of) to download a specific sysmon version.